src/sentry/options/defaults.py

```diff
@@ -3449,3 +3449,6 @@
     default=False,
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+
+# Whether the new objectstore implementation is being used for attachments
+register("objectstore.enable_for.attachments", default=0.0, flags=FLAG_AUTOMATOR_MODIFIABLE)
```

Comment:
This is useful for a gradual rollout. Before we get there, we will want to list specific organization IDs. I think we'll need both options, as a result.